readme.md

```diff
@@ -2,6 +2,11 @@
 
 > Regular expression for matching URLs
 
+## Source
+
+Forked from [url-regex](https://github.com/kevva/url-regex) so we can improve the compability with IE11 or other browsers.
+Needed for consuming it from surface hubs or windows app.
```

Comment:
Why we need this for the hubs and windows app ?

Reply:
url-regex was imported as a regular dependency (meaning that it would not run through Babel), and it was recently upgraded because of security reasons. The thing is that in the upgrade, some non-ie11 compliant features were introduced (like arrow functions and const declarations) and it broke our windows apps.